src/pages/Admin/Room/Edit/RoomEdit.tsx

```diff
@@ -134,24 +54,96 @@ const EditRoom: React.FC<EditRoomProps> = ({
 
       <Modal.Body>
         <Form onSubmit={handleSubmit(onSubmit)}>
-          {renderNameInput()}
-          {renderUrlInput()}
-          {renderDescriptionInput()}
-          {renderLogoInput()}
-
-          {roomTemplate?.customInputs?.map((input) => renderCustomInput(input))}
-
-          {renderTemplateSelect()}
+          <Form.Row>
+            <div className="input-wrapper">
+              <Form.Label>Name your room</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="title"
+                placeholder="Room name"
+                custom
+              />
+            </div>
+          </Form.Row>
+
+          <Form.Row>
+            <div className="input-wrapper">
+              <Form.Label>The room url</Form.Label>
+              <Form.Control
+                type="text"
+                ref={register}
+                name="url"
+                placeholder="Room url"
+                custom
+              />
+            </div>
+          </Form.Row>
```

Comment:
This block seems to differ from how it was previously defined in `renderUrlInput`. What was the reasoning for the differences?

Reply:
As an example, there was no `Form.Row` wrapping this before.